conda/cli/main.py

```diff
@@ -136,21 +136,6 @@ def main():
         logging.disable(logging.NOTSET)
         logging.basicConfig(level=logging.DEBUG)
 
-    if (not main_init.is_initialized() and
-        'init' not in sys.argv and 'info' not in sys.argv):
-        if hasattr(args, 'name') and hasattr(args, 'prefix'):
-            import conda.config as config
-            from conda.cli import common
-            if common.get_prefix(args) == config.root_dir:
-                sys.exit("""\
-Error: This installation of conda is not initialized. Use 'conda create -n
-envname' to create a conda environment and 'source activate envname' to
-activate it.
-
-# Note that pip installing conda is not the recommended way for setting up your
-# system.  The recommended way for setting up a conda system is by installing
-# Miniconda, see: http://repo.continuum.io/miniconda/index.html""")
```

Comment:
@ilanschnell Going to need your input here.  Why was this ever implemented in the first place? There weren't any tests that failed *because of* its removal.